apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx

```diff
@@ -19,7 +30,34 @@ const Title = styled(Text)`
 const Description = styled(Text)`
   font-size: 13px;
 `;
-export function AccountSubHeaderDrawer({ isOpen, closeDrawer, family, team }: Props) {
+
+type TeamLinkProps = {
+  team: string;
+  teamLink: string;
+};
+
+const TeamLink = ({ team, teamLink }: TeamLinkProps) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Description>{t("account.subHeader.drawer.descriptionLink.integration")}</Description>{" "}
+      <Link
+        verticalAlign={"bottom"}
+        onClick={() => openURL(teamLink)}
```

Comment:
can you move this to a handler please ?

Reply:
Done in https://github.com/LedgerHQ/ledger-live/pull/8028/commits/b804236dd4d8d8f24b41a6defeb592fe84d8172b 👍